test/Microsoft.SqlTools.ServiceLayer.UnitTests/LanguageServer/SqlCompletionItemTests.cs

```diff
@@ -13,303 +14,6 @@ namespace Microsoft.SqlTools.ServiceLayer.UnitTests.LanguageServer
 {
     public class SqlCompletionItemTests
     {
-        private static readonly string[] ReservedWords = new string[]
```

Comment:
I don't know why they'd be different. My guess is that these weren't kept up to date with changes made to the original list - which is part of the reason I'm deleting it.

I don't know of a good reason to keep a separate list since we want these tests to be checking against that list of reserved words anyways.